src/main/java/wolox/training/models/Book.java

```diff
@@ -70,6 +71,10 @@ public Long getId() {
         return id;
     }
 
+    public void setId(Long id) {
```

Comment:
We can avoid using a setter for the id, why dont use a custom constructor instead?